src/tests/views/Components/Controls/Switch.test.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import { mount, render } from 'enzyme';
+import theme from 'views/App/theme';
+import 'jest-styled-components';
+import Switch from '../../../../views/Components/Controls/Switch';
```

Comment:
`import Switch from '../../../../views/Components/Controls/Switch';` should be `import Switch from 'views/Components/Controls/Switch';`